instant-xml/src/impls.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{DeserializeXml, Error, FromXml, Visitor};
+use std::str::FromStr;
+
+struct BoolVisitor;
+
+impl<'de> Visitor<'de> for BoolVisitor {
+    type Value = bool;
+
+    fn visit_str<'a>(self, value: &str) -> Result<Self::Value, Error> {
+        Ok(FromStr::from_str(value).unwrap())
+    }
+}
+
+impl<'xml> FromXml<'xml> for bool {
+    fn deserialize<D>(deserializer: &mut D) -> Result<Self, Error>
```

Comment:
I don't think we need a generic `D` here?

Reply:
This would require to change it everywhere to fixed Deserializer including the Visitor trait, I don't know if we want to do this this way? I don't have a strong opinion here.

Reply:
Yes, serde abstracts over this because it implements deserialization across many different formats, but for this crate we're just doing XML.

Reply:
done